cmd/manager/main.go

```diff
@@ -161,7 +163,11 @@ func main() {
 	}
 
 	// Create Service object to expose the metrics port.
-	_, err = metrics.ExposeMetricsPort(ctx, metricsPort)
+	servicePorts := []v1.ServicePort{
+		{Port: metricsPort, Name: metrics.OperatorPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: metricsPort}},
+		// {Port: operatorMetricsPort, Name: metrics.CRPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: operatorMetricsPort}},
```

Comment:
what's the commented out part for?

Reply:
I'll delete it. It was in the upgrade doc: https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#v09x